package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dohnut",
-  "version": "4.6.0",
+  "version": "4.5.3",
```

Comment:
🤔

Reply:
wait wut

Reply:
did I just checkout the wrong hash? . . .

Reply:
And that's why I am an `commit --amend` & `push --force` addict. 😅